occupancy-quickstart/src/actions/userDefinedFunctions/multiplemotionsensors.js

```diff
@@ -0,0 +1,95 @@
+//        - dataType: Motion
+//          hardwareId: SAMPLE_SENSOR_MOTION_THREE
+// 3) Add these additional motion sensors to the device-connectivity project's appSettings.json. For example:
+//    "Sensors": [{
+//      "DataType": "Motion",
+//      "HardwareId": "SAMPLE_SENSOR_MOTION_ONE"
+//    },{
+//      "DataType": "CarbonDioxide",
+//      "HardwareId": "SAMPLE_SENSOR_CARBONDIOXIDE"
+//    },{
+//      "DataType": "Motion",
+//      "HardwareId": "SAMPLE_SENSOR_MOTION_TWO"
+//    },{
+//      "DataType": "Motion",
+//      "HardwareId": "SAMPLE_SENSOR_MOTION_THREE"
+//    }]
+// 
+// 
+
+
+var motionType = "Motion";
+var spaceAvailFresh = "AvailableAndFresh";
+
+function process(telemetry, executionContext) {
+    try {
+       // Log SensorId and Message
+       log(`Sensor ID: ${telemetry.SensorId}. `);
+       log(`Sensor value: ${JSON.stringify(telemetry.Message)}.`);
+
+       // Get sensor metadata
+       var sensor = getSensorMetadata(telemetry.SensorId);
+       
+       // Retrieve the sensor reading
+       var parseReading = JSON.parse(telemetry.Message);
+       
+       // Set the sensor reading as the current value for the sensor.
+       setSensorValue(telemetry.SensorId, sensor.DataType, parseReading.SensorValue);
+       
+       // Get parent space
+       var parentSpace = sensor.Space();
+       
+       // Get all children sensors of the parent space
+       var allSensors = parentSpace.ChildSensors();
+
+       // Get all motion sensors of the parent space
+       var motionSensors = allSensors.filter(function(item){
+         return item.DataType === motionType;
+       });
+
+       // Is any of the motion sensors detecting motion?
+       var motionDetected = motionSensors.find(function(element) {
+         return element.Value().Value.toLowerCase() === "true";
+       });
+
+       // Set OccupancyStatus for the space
+       var occupied = "Room is occupied";
+       var empty = "Room is empty!";
+                
+       if (motionDetected != undefined)
+       {
+         // Motion is detected by at least one sensor
+         log(`${occupied}. Motion Detected: ${motionDetected}. Sensors: ${motionSensors}.`);
+         setSpaceValue(parentSpace.Id, spaceAvailFresh, occupied);
+       }
+       else {
+         // No motion detected by any sensor
+         log(`${empty}. Motion Not Detected: ${motionDetected}. Sensors: ${motionSensors}.`);
+         setSpaceValue(parentSpace.Id, spaceAvailFresh, empty);
+         // You could try creating a Logic App mail notification in case none of the motion sensors
+         // detect anything. In that case, uncomment the following line:
+         //  parentSpace.Notify(JSON.stringify(empty));
+       }
+   }
+   catch (error)
+   {
+      var errormsg = `An error has occurred processing the UDF Error: ${error.name} Message ${error.message}.`;
+      log(errormsg);
+      setSpaceValue(parentSpace.Id, spaceAvailFresh, errormsg);
```

Comment:
why setting the space spaceAvailFresh? This sample doesn't have anything to do with fresh ..we are only detecting motion, right?